src/inmanta/graphql/schema.py

```
@@ -526,6 +566,86 @@ def do_required_resource_joins(
     return stmt
 
 
+def encode_cursor(cursor: str) -> str:
+    """
+    :param cursor: The cursor received from sqlakeyset without direction information ('<'/'>').
+    :return: The base64 encoded cursor in str form.
+    """
+    return base64.b64encode(f"{EDGE_CURSOR_PREFIX}{cursor}".encode()).decode()
+
+
+def decode_cursor(cursor: str) -> str:
+    """
+    :param cursor: The cursor received as an argument from the user.
+        Expected in the format that Edge.resolve_edge returns (prefixed with `EDGE_CURSOR_PREFIX`).
+    :return: The decoded cursor in str form.
+    """
+    return base64.b64decode(cursor).decode().split(EDGE_CURSOR_PREFIX)[1]
```

if the string is not in here, I would raise an exception